app/controllers/Application.scala

```diff
@@ -153,8 +154,7 @@ class Application(
   }
 }
 
-case class VisitToken(value: String) extends AnyVal
-object VisitToken {
-  def apply(rand: SecureRandom): VisitToken =
-    new VisitToken(s"${rand.nextLong.toHexString}")
+object CreateScopeToken {
```

Comment:
the case class is now in the support models.  (Even though at this point it's yet to be serialised, sent to the client side, and sent back again for deserialising.)